src/style.css

```diff
@@ -1,3 +1,68 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500&display=swap");
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.navigation__wraper {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__logo {
+  align-self: center;
+  justify-content: center;
+  height: 40px;
```

Comment:
All those styles doesn't do anything. You should set `height` or `width` directly on image.